WalletWasabi.Fluent/Behaviors/RandomizedWorldPointsBehavior.cs

```diff
@@ -146,14 +146,14 @@ protected override void OnAttached()
 		{
 			base.OnAttached();
 
-			if (AssociatedObject is null)
-			{
-				return;
-			}
-
 			Dispatcher.UIThread.Post(
 				() =>
 				{
+					if (AssociatedObject?.Children is null || AssociatedObject?.Children?.Count == 0)
```

Comment:
If we get to the or part of the if clause then we don't need to check for null, To fix the null warning in line 157:

```suggestion
					if (AssociatedObject?.Children is null || AssociatedObject.Children.Count == 0)
```